.ci/310.yaml

```
@@ -15,6 +15,7 @@ dependencies:
   - shapely
   - spaghetti
   - tqdm
+  - pointpats=2.3
```

Let's remove the pin here (and all other `.ci/` environments ***except*** `.ci/38-MIN.yaml`):

```suggestion
  - pointpats
```